src/sparseml/export/export_data.py

```diff
@@ -196,6 +198,28 @@ def run_inference_with_dict_data(
     return inputs, labels, output
 
 
+def _unnest_past_key_values(output_vals: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Not sure I'm following what this is for, can you give a quick example of what output_vals looks like before and after this function? Why is it that output_vals contains these `past_key_values` in the first place?

Reply:
This is removed now all together